client/src/Utils/Validations/Validator.test.ts

```diff
@@ -0,0 +1,85 @@
+import Validator from "./Validator";
+
+
+describe('PhoneValidation Tests', () => {
```

Comment:
If the file is for validation testing in general maybe the describe should be 'validation tests'

Reply:
you are right.